packages/node/__tests__/index.test.ts

```diff
@@ -10,6 +10,11 @@ import config from '../src/config';
 import pkg from '../package.json';
 import { expressMiddleware } from '../src';
 import { ServerResponse } from 'http';
+import FormData from 'form-data';
```

Comment:
For what it's worth,`form-data` isn't spec-compliant and it's given me lots of headaches (https://github.com/readmeio/httpsnippet/pull/1) in the past.

> * This fixes the implementation of [form-data](https://www.npmjs.com/package/form-data) in `multipart/form-data` content types in order to accommodate library utilization within the browser where the native `FormData` component is used.
>   * Key differences between `form-data` and `FormData` that was causing the library to fail in the browser were two issues:
>     * The API for `FormData.append` has three arguments, and the third should only be present if the second is a `Blob` or `USVString`. It is never an object, as `form-data` requires it to be.
>     * `FormData.pipe()` isn't a function.

`node-fetch` folks recommend [formdata-polyfill]([https://www.npmjs.com/package/formdata-polyfill) these days. 

But that said, what you're using it here for should be fine.

Reply:
that's *great* to know, and I'll make a note of it, but yeah prob not an issue for these tests.